src/parser.rs

```diff
@@ -952,6 +953,11 @@ impl Library {
             "attribute" => parser.ignore_element(),
             _ => Err(parser.unexpected_element(elem)),
         })?;
+        // The last argument of a callback is ALWAYS user data, so it has to be marked as such
+        // in case it's missing.
```

Comment:
It will always be missing unless the annotations are wrong :)

Reply:
Well, apparently it was always present until now...